code/modules/surgery/organs/organ_internal.dm

```diff
@@ -394,3 +428,11 @@
 	SIGNAL_HANDLER
 	REMOVE_TRAIT(src, TRAIT_ORGAN_INSERTED_WHILE_DEAD, "[UID()]")
 	UnregisterSignal(owner, COMSIG_LIVING_DEFIBBED)
+
+/obj/item/organ/internal/proc/Start()
+	return
```

Comment:
Is this used by anything? If so, please change the name to `start()` and add a comment explaining what it's supposed to be used for.